Operations to perform:
  Apply all migrations: accounts, admin, auth, contenttypes, gallery, group, photo, photosets, sessions
Running migrations:
Not implemented alter command for SQL ALTER TABLE "accounts_account" ADD COLUMN "count_groups" long DEFAULT %(0)s NOT NULL
  Applying accounts.0002_auto_20210531_2147...Traceback (most recent call last):
  File "/home/ziyadmin/Desktop/Deployment/Software-Project/flickr-backend/venv/lib/python3.8/site-packages/djongo/cursor.py", line 51, in execute
    self.result = Query(
  File "/home/ziyadmin/Desktop/Deployment/Software-Project/flickr-backend/venv/lib/python3.8/site-packages/djongo/sql2mongo/query.py", line 783, in __init__
    self._query = self.parse()
  File "/home/ziyadmin/Desktop/Deployment/Software-Project/flickr-backend/venv/lib/python3.8/site-packages/djongo/sql2mongo/query.py", line 875, in parse
    raise e
  File "/home/ziyadmin/Desktop/Deployment/Software-Project/flickr-backend/venv/lib/python3.8/site-packages/djongo/sql2mongo/query.py", line 856, in parse
    return handler(self, statement)
  File "/home/ziyadmin/Desktop/Deployment/Software-Project/flickr-backend/venv/lib/python3.8/site-packages/djongo/sql2mongo/query.py", line 888, in _alter
    query = AlterQuery(self.db, self.connection_properties, sm, self._params)
  File "/home/ziyadmin/Desktop/Deployment/Software-Project/flickr-backend/venv/lib/python3.8/site-packages/djongo/sql2mongo/query.py", line 425, in __init__
    super().__init__(*args)
  File "/home/ziyadmin/Desktop/Deployment/Software-Project/flickr-backend/venv/lib/python3.8/site-packages/djongo/sql2mongo/query.py", line 84, in __init__
    super().__init__(*args)
  File "/home/ziyadmin/Desktop/Deployment/Software-Project/flickr-backend/venv/lib/python3.8/site-packages/djongo/sql2mongo/query.py", line 62, in __init__
    self.parse()
  File "/home/ziyadmin/Desktop/Deployment/Software-Project/flickr-backend/venv/lib/python3.8/site-packages/djongo/sql2mongo/query.py", line 435, in parse
    self._add(statement)
  File "/home/ziyadmin/Desktop/Deployment/Software-Project/flickr-backend/venv/lib/python3.8/site-packages/djongo/sql2mongo/query.py", line 598, in _add
    raise SQLDecodeError(err_key=tok.value,
djongo.exceptions.SQLDecodeError: 

	Keyword: long
	Sub SQL: ALTER TABLE "accounts_account" ADD COLUMN "count_groups" long DEFAULT %(0)s NOT NULL
	FAILED SQL: ('ALTER TABLE "accounts_account" ADD COLUMN "count_groups" long DEFAULT %(0)s NOT NULL',)
	Params: ([0],)
	Version: 1.3.4

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/home/ziyadmin/Desktop/Deployment/Software-Project/flickr-backend/venv/lib/python3.8/site-packages/django/db/backends/utils.py", line 86, in _execute
    return self.cursor.execute(sql, params)
  File "/home/ziyadmin/Desktop/Deployment/Software-Project/flickr-backend/venv/lib/python3.8/site-packages/djongo/cursor.py", line 59, in execute
    raise db_exe from e
djongo.database.DatabaseError

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "manage.py", line 21, in <module>
    main()
  File "manage.py", line 17, in main
    execute_from_command_line(sys.argv)
  File "/home/ziyadmin/Desktop/Deployment/Software-Project/flickr-backend/venv/lib/python3.8/site-packages/django/core/management/__init__.py", line 401, in execute_from_command_line
    utility.execute()
  File "/home/ziyadmin/Desktop/Deployment/Software-Project/flickr-backend/venv/lib/python3.8/site-packages/django/core/management/__init__.py", line 395, in execute
    self.fetch_command(subcommand).run_from_argv(self.argv)
  File "/home/ziyadmin/Desktop/Deployment/Software-Project/flickr-backend/venv/lib/python3.8/site-packages/django/core/management/base.py", line 328, in run_from_argv
    self.execute(*args, **cmd_options)
  File "/home/ziyadmin/Desktop/Deployment/Software-Project/flickr-backend/venv/lib/python3.8/site-packages/django/core/management/base.py", line 369, in execute
    output = self.handle(*args, **options)
  File "/home/ziyadmin/Desktop/Deployment/Software-Project/flickr-backend/venv/lib/python3.8/site-packages/django/core/management/base.py", line 83, in wrapped
    res = handle_func(*args, **kwargs)
  File "/home/ziyadmin/Desktop/Deployment/Software-Project/flickr-backend/venv/lib/python3.8/site-packages/django/core/management/commands/migrate.py", line 231, in handle
    post_migrate_state = executor.migrate(
  File "/home/ziyadmin/Desktop/Deployment/Software-Project/flickr-backend/venv/lib/python3.8/site-packages/django/db/migrations/executor.py", line 117, in migrate
    state = self._migrate_all_forwards(state, plan, full_plan, fake=fake, fake_initial=fake_initial)
  File "/home/ziyadmin/Desktop/Deployment/Software-Project/flickr-backend/venv/lib/python3.8/site-packages/django/db/migrations/executor.py", line 147, in _migrate_all_forwards
    state = self.apply_migration(state, migration, fake=fake, fake_initial=fake_initial)
  File "/home/ziyadmin/Desktop/Deployment/Software-Project/flickr-backend/venv/lib/python3.8/site-packages/django/db/migrations/executor.py", line 245, in apply_migration
    state = migration.apply(state, schema_editor)
  File "/home/ziyadmin/Desktop/Deployment/Software-Project/flickr-backend/venv/lib/python3.8/site-packages/django/db/migrations/migration.py", line 124, in apply
    operation.database_forwards(self.app_label, schema_editor, old_state, project_state)
  File "/home/ziyadmin/Desktop/Deployment/Software-Project/flickr-backend/venv/lib/python3.8/site-packages/django/db/migrations/operations/fields.py", line 110, in database_forwards
    schema_editor.add_field(
  File "/home/ziyadmin/Desktop/Deployment/Software-Project/flickr-backend/venv/lib/python3.8/site-packages/django/db/backends/base/schema.py", line 480, in add_field
    self.execute(sql, params)
  File "/home/ziyadmin/Desktop/Deployment/Software-Project/flickr-backend/venv/lib/python3.8/site-packages/django/db/backends/base/schema.py", line 142, in execute
    cursor.execute(sql, params)
  File "/home/ziyadmin/Desktop/Deployment/Software-Project/flickr-backend/venv/lib/python3.8/site-packages/django/db/backends/utils.py", line 100, in execute
    return super().execute(sql, params)
  File "/home/ziyadmin/Desktop/Deployment/Software-Project/flickr-backend/venv/lib/python3.8/site-packages/django/db/backends/utils.py", line 68, in execute
    return self._execute_with_wrappers(sql, params, many=False, executor=self._execute)
  File "/home/ziyadmin/Desktop/Deployment/Software-Project/flickr-backend/venv/lib/python3.8/site-packages/django/db/backends/utils.py", line 77, in _execute_with_wrappers
    return executor(sql, params, many, context)
  File "/home/ziyadmin/Desktop/Deployment/Software-Project/flickr-backend/venv/lib/python3.8/site-packages/django/db/backends/utils.py", line 86, in _execute
    return self.cursor.execute(sql, params)
  File "/home/ziyadmin/Desktop/Deployment/Software-Project/flickr-backend/venv/lib/python3.8/site-packages/django/db/utils.py", line 90, in __exit__
    raise dj_exc_value.with_traceback(traceback) from exc_value
  File "/home/ziyadmin/Desktop/Deployment/Software-Project/flickr-backend/venv/lib/python3.8/site-packages/django/db/backends/utils.py", line 86, in _execute
    return self.cursor.execute(sql, params)
  File "/home/ziyadmin/Desktop/Deployment/Software-Project/flickr-backend/venv/lib/python3.8/site-packages/djongo/cursor.py", line 59, in execute
    raise db_exe from e
